lib/mpl_toolkits/tests/test_axisartist_floating_axes.py

```diff
@@ -18,7 +18,7 @@ def test_subplot():
     fig.add_subplot(ax)
 
 
-@image_comparison(['curvelinear3.png'], style='default', tol=0.01)
+@image_comparison(['curvelinear3.png'], style='default', tol=5)
```

Comment:
Put a comment to reduce the tolerance when the image gets regenerated?

Reply:
done